code/modules/cargo/exports/sheets.dm

```diff
@@ -96,13 +96,13 @@
 // Weird Stuff
 
 /datum/export/stack/abductor
-	cost = 1000
+	cost = 2000
 	message = "of alien alloy"
 	export_types = list(/obj/item/stack/sheet/mineral/abductor)
 
 /datum/export/stack/adamantine
 	unit_name = "bar"
-	cost = 500
+	cost = 5000
```

Comment:
5000 is far too much considering adamantine factory(industrial adamantine), reproductive adamantine and stabilized metal crossbreeds allow  bulk production. I'd suggest 1000 or 1500.

Reply:
you realize the price on exports drops every time a piece is exported right